### PR TITLE
FEC-12185 Exposed AdResponse APIs on `MediaInfoUtils`

### PR DESCRIPTION
**Pull request by GouravSna**
- Updated README
- Exposed API to use ad response for ad config

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -198,7 +198,9 @@ var pendingResult: PendingResult<RemoteMediaClient.MediaChannelResult>? = null
 val loadOptions = MediaLoadOptions.Builder().setAutoplay(true).setPlayPosition(0L).build()
 
 ```
-> **Important: In case for Live Media, `setPlayPosition` should be passed `-1`**
+> **NOTE: In case for Live Media, `setPlayPosition` should be passed `MediaInfoUtils.LIVE_EDGE` or `-1000L`**.
```
Comment (GouravSna) [inline README.md]: - Check with -1000L value

Comment (GouravSna) [inline README.md]: Check with -1000L value

**Review by giladna — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (GouravSna) [reply README.md]: Done. Changed to -1L.

Comment (GouravSna) [reply README.md]: done.

**Pull request merged**